cypress/integration/sidebar.js

```diff
@@ -0,0 +1,7 @@
+describe('Sidebar', () => {
```

Comment:
Possibly worth adding whether the Default dashboard appears in the sidebar?

Reply:
done!